debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/BinlogReaderIT.java

```diff
@@ -440,6 +440,38 @@ public void shouldHandleMySQLTimeCorrectly() throws Exception {
         assertThat(c5Time).isEqualTo(Duration.ofHours(-838).minusMinutes(59).minusSeconds(58).minusNanos(999999000));
     }
 
+    @Test
+    public void shouldConsumeAllEventsFromDatabaseWithSkippedOperations() throws Exception {
+        // Define configuration that will ignore all events from MySQL source.
+        config = simpleConfig()
+                .with(MySqlConnectorConfig.SKIPPED_OPERATIONS, "i")
```

Comment:
The operation name is `c` not `i`.

Reply:
Other than that, it'd be nice to make sure that another event *is* received after skipped ones. So say you test skipping inserts, there should be an assertion for present of a subsequent update or delete of that record.